.gitlab-ci.yml

```diff
@@ -1,5 +1,28 @@
 image: docker:dind
 
+############################################################
+# Section for security tool includes
+############################################################
+include:
+  - project: "scurity/gitlabci"
+    ref: master
+    file: "/.gitlab-ci-config-ssh.yml"
```

Comment:
The ssh config is in the scurity folder now. I'm not sure if this is the correct place or not. Isn't this for settings common to multiple repos? If so, then wouldn't this yml file be better suited somewhere else? in APIGOV, we have a folder called beano-cicd where we were intending on putting some of the settings that are common to our agent repos. Maybe this would be a better location for the ssh settings, since there are specific references to the sdk?

Just a thought. If you think it is getter suited where it is, then that's OK with me.

Reply:
I didnt realise there was a beano-cicd common gitlab. It would probably be better suited there, I was just trying to find a quick solution and didnt want to go creating a new repo. 
I'm perfectly happy for it to be moved.

Reply:
OK. If it's OK, I think I'll merge this for now as it is, and then move it when I continue on with the other things that need to be moved out of the SDK repo. This way, it will get the pipeline as it currently stands, moving along.